plugin-code/src/main/resources/template/macros/scenario.ftl

```diff
@@ -105,22 +105,63 @@ limitations under the License.
 
 <#macro attachments step>
     <#if step.embeddings??>
+    	<#assign index=1>
         <#list step.embeddings as attachment>
-            <div class="row w-100 p-3 m-0 scenarioAttachment">
-                <div class="w-100 text-center m-auto">
-                    <#if attachment.image>
-                        <a class="grouped_elements" rel="images"
-                           href="attachments/${attachment.filename}">
-                            <img src="attachments/${attachment.filename}"
-                                 style="max-width: 100%"/>
-                        </a>
-                    <#else>
-                        ${attachment.data?html}
-                    </#if>
-                </div>
-            </div>
+                <div class="row w-100 p-3 m-0 scenarioAttachment">
+                	<div class="w-100 text-left m-auto border border-dark">             
+                    	<#if attachment.image>
```

Comment:
Personally, I am not a fan of making attachments collapsible (plus it looks inconsistent with the existing design). Also, this behaviour was not described in the description of issue #97 which was just about correctly showing the attachments in AfterHooks.

Reply:
I added this behavior to reduce the scrolling overhead.(Currently, it is inconsistent with the existing design, we will have to work on it)
 
As per my opinion, lots of scrolling is already required to see the scenario steps result and one image attachments. In many cases, there could be requirements to attach scenario execution logs, uploaded files, downloaded files, additional reporting parameters and attachments. This will increase scrolling overhead too much.

Reply:
Agreed. Should I keep titles for attachments like "Attachment 1 (html)" or remove them too?

Reply:
I think we can remove them.

Reply:
removed attachments collapsible behavior